README.md

```diff
@@ -0,0 +1,32 @@
+# Masters-way app 
+
+Multi-user application for tracking the educational process.
+
+### Modules
+* docs - project documentation
+* mw-webapp - front end for clients
+___
+
+### Requirements: 
+* pnpm: 7.25.0
```

Comment:
usually platform version is more important than package manager version

Reply:
Add node version to requirements